cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -129,7 +129,7 @@ phases:
                 [[ $OS =~ rhel ]] && yum versionlock redhat-release
               else
                 apt-get -y install linux-headers-$(uname -r) linux-modules-extra-$(uname -r)
-                apt-mark hold linux-aws* linux-base* linux-headers* linux-image* linux-modules-extra*
+                apt-mark hold linux-aws-$(uname -r)* linux-base-$(uname -r)* linux-headers-$(uname -r)* linux-image-$(uname -r)* linux-modules-extra-$(uname -r)*
```

Comment:
[minor] We execute `uname -r` in many places (not only here, but across the whole component). 
What about having it into a variable rather than calling the command every time?